client/src/js/base/SelectBox.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import { Box } from "./Box";
+
+export const SelectBox = styled(Box)`
+    border: 1px ${props => (props.active ? props.theme.color.primary : props.theme.color.greyLight)} solid;
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
```

Comment:
Is this necessary? The default value for `display` is `block`, which I think will have the same effect.